Comments/ClientApp/src/components/Document/Document.js

```diff
@@ -440,6 +448,11 @@ export class Document extends Component<PropsType, StateType> {
 					<div className="grid">
 						<div data-g="12">
 							<BreadCrumbsWithRouter links={this.state.consultationData.breadcrumbs}/>
+							{!this.state.consultationIsOpen &&
```

Comment:
why not just use this.state.consultationData.consultationState.consultationIsOpen and drop all of the above?

Reply:
change made